src/components/container/BloodPressureVisualization/BloodPressureVisualization.tsx

```diff
@@ -0,0 +1,257 @@
+import "./BloodPressureVisualization.css";
+import { useEffect, useState } from "react";
+import { debounce } from "lodash";
+import MyDataHelps, { Guid, SurveyAnswer, SurveyAnswersPage, SurveyAnswersQuery } from "@careevolution/mydatahelps-js";
+import plus from './plus.svg';
+import React from "react";
+import { Button, DateRangeNavigator, LoadingIndicator } from "../../presentational";
+import DumbbellChart from "../../presentational/DumbbellChart";
+import { Dumbbell, ClosedInterval, Axis, DataPoint, DumbbellClass } from "../../presentational/DumbbellChart/DumbbellChart";
+import { previewSurveyData } from './BloodPressureVisualization.previewdata'
+import { addDays, format, isEqual, isMonday, nextSunday, parseISO, previousMonday, startOfDay } from "date-fns";
+
+export type DataFaucet = "PreviewSurvey" | "Survey";
+
+export interface BloodPressureVisualizationProps {
```

Comment:
This same concept will be needed for the Blood Pressure Metrics component. Is there a best practice for sharing these? I didn't see existing components with this same situation

Reply:
I don't think that should be a separate component?  Seems like it should be a part of this one?

Reply:
Maybe not understanding what you are asking though